src/credentials/status/agent-revocation.ts

```diff
@@ -63,15 +61,23 @@ function buildRevocationMessageRequest(
   };
 }
 
-function toRevocationStatus(revocationResponse: RevocationStatusResponseMessage): RevocationStatus {
+function toRevocationStatus(revocationResponse: any): RevocationStatus {
   if (!revocationResponse.body?.mtp) {
     throw new Error('Revocation status response does not contain mtp');
   }
   if (!revocationResponse.body?.issuer) {
     throw new Error('Revocation status response does not contain issuer');
   }
-  return {
+  const rs = {
     mtp: revocationResponse.body.mtp,
     issuer: revocationResponse.body.issuer
   };
+  if (revocationResponse.body.mtp.node_aux) {
+    rs.mtp.nodeAux = {
+      key: newHashFromBigInt(BigInt(revocationResponse.body.mtp.node_aux.key)),
+      value: newHashFromBigInt(BigInt(revocationResponse.body.mtp.node_aux.value))
+    };
+  }
+
```

Comment:
perhaps, you should use the code

```
const p = new Proof();
    p.existence = this.mtp.existence;
    if (this.mtp.node_aux) {
      p.nodeAux = {
        key: newHashFromBigInt(BigInt(this.mtp.node_aux.key)),
        value: newHashFromBigInt(BigInt(this.mtp.node_aux.value))
      };
    }
    const s = this.mtp.siblings.map((s) => newHashFromBigInt(BigInt(s)));

    p.siblings = [];
    p.depth = s.length;

    for (let lvl = 0; lvl < s.length; lvl++) {
      if (s[lvl].bigInt() !== BigInt(0)) {
        setBitBigEndian(p.notEmpties, lvl);
        p.siblings.push(s[lvl]);
      }
    }
    return {
      mtp: p,
      issuer: this.issuer
    };
```
you should reuse RevocationStatusDTO

Reply:
done